### ISSUE TITLE
Add documentation on accessibility overlays and plugins

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/docs#315]
From Slack channel: https://localgovdrupal.slack.com/archives/C020SJRQX2M/p1739264234642729

**Comment by JonathanDHolden** [comment]
We should link here https://overlayfactsheet.com/en/ and maybe encourage people to sign it too.
